webgrid/templates/grid_footer.html

```diff
@@ -2,6 +2,10 @@
     {% if not grid.hide_excel_link %}
         <p><a class="export-link" href="{{ renderer.export_url() }}">export to Excel</a></p>
     {% endif %}
+    
+    {% if not grid.hide_csv_link %}
```

Comment:
@guruofgentoo not sure if we want a flag for showing the csv link or if we want to make the excel link more dynamic and manipulate `default_spreadsheet_format` to control how we are exporting from the grid

Reply:
I would rather have a single flag to control show/hide for export links in general, and then show links based on the allowed renderers for the grid.